windows_scripts/! Install.bat

```diff
@@ -0,0 +1,17 @@
+:main
+    @echo off
+	
+	for %%d in (%~dp0..) do set ParentDirectory=%%~fd
+	
+	SET /P INSTALLMATHPLOTLIB="Install mathplotlib & tensorboard for stats (Y/[N])?: "
+	IF /I "%INSTALLMATHPLOTLIB%" == "Y" call :InstallMathPlotLibTensorflow
+	
+	pip install "%ParentDirectory%"
+	
+	pause
+	
+	goto :eof
+
+:InstallMathPlotLibTensorflow

```

Comment:
just noticed this, please fix the typo `mathplotlib -> matplotlib`

Reply:
oops :face_with_spiral_eyes: